core/types/accounts/account_test.go

```diff
@@ -277,6 +278,33 @@ func testIncarnationWithNoIncarnation(t *testing.T) {
 
 }
 
+func testIncarnationWithInvalidEncodedAccount(t *testing.T) {
```

Comment:
I think if it starts with the lowercase letter, it does not automatically get executed, please check. I think it needs to start with capital "T"